tests/components/matter/fixtures/nodes/device_diagnostics.json

```diff
@@ -0,0 +1,4223 @@
+{
+  "node_id": 5,
+  "date_commissioned": "2023-01-16T21:07:57.508440",
+  "last_interview": "2023-01-16T21:07:57.508448",
+  "interview_version": 1,
+  "attributes": {
+    "0/4/0": {
```

Comment:
Can we collapse this? the whole dict value of this is not needed. That's just for lookup. 

- Node id is in parent
- endpoint ID, cluster ID and attribute ID are in the key.
- Cluster/Attribute type/name are not needed as we have ID

Reply:
That's a question for the server. I don't think we should adjust anything here. We include what we get from the server.

Personally I like the verbose human readable values in the of cluster and attribute type, and think we should keep them.